modules/Bio/EnsEMBL/Variation/DBSQL/BaseAnnotationAdaptor.pm

```diff
@@ -279,5 +293,12 @@ sub _ftp_file_exists {
   }
 }
 
+sub _check_file_over_http {
+  my ($self, $uri) = @_;
+  my $ua = HTTP::Tiny->new();
+  my $response = $ua->head($uri);
+
+  return !!$response->{success};
```

Comment:
one ! too many

Reply:
I mostly blindly copied this from https://metacpan.org/release/URL-Exists/source/lib/URL/Exists.pm 😄 
But now that you mention it, it looks like there is no need for the `!!` at all. The subroutine should return true (well, `1`) if the request is successful and false (`0`) if it isn't. I believe that's what the `$response->{success}` field contains anyway ([link to docs](https://metacpan.org/pod/HTTP::Tiny))

Reply:
I removed the redundant double bang. `$response->{success}` is either 1 or 0.